packages/instant/src/components/amount_input.tsx

```diff
@@ -10,10 +10,13 @@ export interface AmountInputProps {
     fontColor?: ColorOption;
     fontSize?: string;
     value?: BigNumber;
-    onChange?: (value?: BigNumber) => void;
+    onChange: (value?: BigNumber) => void;
```

Comment:
just curious -- why move this from being optional to being required but having a default no-op? is this a pattern we want to follow for callbacks?

Reply:
So yea this is a bit confusing. @types/react and ts > 3.0.0 added understanding of `defaultProps`. So the way you think about the `Props` interface is that it's the internal interface (in this case, callback is not optional). However, if you have a default value the compiler knows that you actually don't have to pass the prop in (ie. the external interface is a function of internal interface + defaultProps). 

So in practice it's not required, has default no-ops, so internally you can assume it's not undefined.

And yea I like this pattern :P